velox/docs/functions/spark/datetime.rst

```diff
@@ -34,6 +34,17 @@ These functions support TIMESTAMP and DATE input types.
 
     SELECT dayofmonth('2009-07-30'); -- 30
 
+.. spark:function:: dayofweek(date/timestamp) -> integer
+
+    Returns the day of the week for date/timestamp (1 = Sunday, 2 = Monday, ..., 7 = Saturday).
+    We can use `dow` as alias for ::
+
+    SELECT dayofweek('2009-07-30'); -- 5
+
+.. function:: dow(x) -> bigint
```

Comment:
This function return type should be int32
```
trait GetDateField extends UnaryExpression with ImplicitCastInputTypes with NullIntolerant {
  val func: Int => Any
  val funcName: String

  override def inputTypes: Seq[AbstractDataType] = Seq(DateType)

  override def dataType: DataType = IntegerType
```

Reply:
Fixed